build.gradle

```diff
@@ -100,12 +111,22 @@ dependencies {
 	testImplementation('org.spockframework:spock-core:2.0-groovy-3.0') {
 		exclude module: 'groovy-all'
 	}
+	testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
 	testImplementation 'io.javalin:javalin:3.13.11'
 	testImplementation 'net.fabricmc:fabric-installer:0.9.0'
 
 	compileOnly 'org.jetbrains:annotations:23.0.0'
 }
 
+configurations.all {
+	if (name == "compileOnly") return
+
+	dependencies.removeIf {
+		// Rely on the kotlin version gradle uses everywhere else
+		group == "org.jetbrains.kotlin"
```

Comment:
You can do `kotlin.stdlib.default.dependency = false` in `gradle.properties` to disable the stdlib dependency added by the Kotlin plugin

Reply:
I will try that, however its needed to compile against, but not be used at runtime/exposed via the pom.

Reply:
Yeah, that's what it should do in this context. Gradle already adds a kotlin stdlib dependency I think?

Reply:
Yeah, that seems to have worked, thanks 👍